webrender/build.rs

```diff
@@ -211,6 +213,7 @@ fn process_glsl_for_spirv(file_path: &Path, file_name: &str) -> Option<PipelineR
     let mut binding = 1; // 0 is reserved for Locals
     let mut in_location = 0;
     let mut out_location = 0;
+    let mut varying_location = MAX_VERTEX_INPUT_ATTRIBUTES; // 0..15 is reserved for vertex `in` and fragment `out` variables
```

Comment:
I think you meant "vertex `out` and fragment `in`"

Reply:
I meant we have 0..15 for in_location (vertex in) and out_location (fragent out), that's why we start the indexing of varying variables from 16. But I will update the comment, to make it clear.

Reply:
wait, why are vertex inputs and fragment outputs related anyway?

Reply:
The fragment output just simply fits in the 0..15 interval, because we only have one or two from them, that's why I mentioned them.